source/getting-started/enterprise-roll-out-checklist.rst

```diff
@@ -0,0 +1,278 @@
+Enterprise Roll Out Checklist
+==============================
+
+This checklist is intended to serve as a guide to Enterprises who are rolling out Mattermost to thousands of users.  It includes preparing for the roll out, rolling out to users, and reviewing the roll out.  
+
+1. Prepare for the Roll Out
+-----------------------
+Much of the preparation work is focused on ensuring the environment is deployed and secured prior to onboarding users. 
+
+a. Define the Roll Out Project
```

Comment:
One consideration is using "1a.", "1b." etc to clarify where they belong to when quickly scanning through. But I'm totally 0/5 and we don't have a precedence for it. Non-blocking